src/styles/global.css

```
@@ -412,6 +412,58 @@ summary,
   background: var(--nd-divider);
 }
 
+/* ミュート絵文字プレースホルダー (#612)。SVG を mask にして currentColor で
+   塗るため、カラーテーマの文字色に追従する */
+._emojiMuted {
+  display: inline-block;
+  height: 1.25em;
+  aspect-ratio: 1;
+  vertical-align: -0.25em;
+  background: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)